src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/22.sql

```diff
@@ -0,0 +1,2721 @@
+
+/*************************************************************************************************
+    Auto-Generated from Sql build task. Do not manually edit it. 
+**************************************************************************************************/
+SET XACT_ABORT ON
+BEGIN TRAN
+IF EXISTS (SELECT *
+           FROM   sys.tables
+           WHERE  name = 'ClaimType')
+    BEGIN
+        ROLLBACK;
+        RETURN;
+    END
+
+
+GO
+INSERT  INTO dbo.SchemaVersion
+VALUES (21, 'started');
```

Comment:
Should this be 22? (I noticed it's set to 22 in `TransactionCheckWithInitialiScript.sql` but not sure why that didn't carry over here)

Reply:
I have updated this with the last commit